src/NUnitEngine/nunit.engine/Services/ResultService.cs

```diff
@@ -27,17 +27,14 @@
 
 namespace NUnit.Engine.Services
 {
-    public class ResultService :
-#if !NETSTANDARD1_3
-        Service, 
-#endif
-        IResultService
+    public class ResultService : Service,  IResultService
     {
-#if NETSTANDARD1_3
+#if NETSTANDARD1_3 || NETSTANDARD2_0
         private readonly string[] BUILT_IN_FORMATS = new string[] { "nunit3", "cases" };
```

Comment:
If we enable XmlTransforms for .NET Standard 2.0, we need to permit 'user' here as well.